src/actions/exchangeActions.js

```diff
@@ -379,3 +384,56 @@ export const getExchangeSupportedAssetsAction = (callback?: () => void) => {
     dispatch(saveDbAction('exchangeSupportedAssets', { exchangeSupportedAssets }, true));
   };
 };
+
+export const getWbtcFeesAction = () => (dispatch: Dispatch) => {
+  fetch('https://lightnode-mainnet.herokuapp.com', {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      id: 67,
+      jsonrpc: '2.0',
+      method: 'ren_queryFees',
+      params: {},
+    }),
+  })
+    .then(res => res.json())
+    .then(res => {
+      dispatch({ type: SET_WBTC_FEES, payload: res.result });
+    })
+    .catch(e => reportErrorLog('Failed to fetch WBTC fees', e));
```

Comment:
Do we have any feedback for user in case of fail?

Reply:
Added handling this in Exchange screen